10-3/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" type="text/css" href="css/style.css">
+		<title>Carousel</title>
+	</head>
+	<body>
+		<div class="container">
+			<div id="carousel">
+				<ul>
+					<li><img src="https://images.pexels.com/photos/106399/pexels-photo-106399.jpeg" alt="A suburban house"></li>
```

Comment:
pexels ma taka opcje, ze mozesz dorzucic do linka obrazka parametry szerokosci i wysokosci, zeby sciagal mniejsza wersje i obrazek nie ladowal sie tak dlugo wyglada to tak:
`https://images.pexels.com/photos/106399/pexels-photo-106399.jpeg?w=600&h=400`